volumina/sliceSelectorHud.py

```diff
@@ -168,7 +168,7 @@ def __init__(self, parentView, backgroundColor, foregroundColor,
         font.setPixelSize(fontSize)
         self.spinBox.setFont(font)
         self.do_draw()
-
+        self._parentView = parentView
     def do_draw(self):
```

Comment:
Similarly, I think we should avoid storing this `parentView` member here.  (But in this case, the previous author forgot to pass `parent` to the base class (see above), so you'll need to fix that.)